new-site/README.md

```diff
@@ -0,0 +1,57 @@
+# Helsinki Design System documentation site
+
+This is the public documentation site of Helsinki Design System ([hds.hel.fi](https://hds.hel.fi/)). It is built with [Gatsby](https://www.gatsbyjs.com/) and it bases on [Gatsby's default starter](https://github.com/gatsbyjs/gatsby-starter-default).
+- General introduction of Helsinki Design System
+- Getting started section for designers and developers
+- Guidelines for accessibility, grid, localisation and more
+- Guidelines for visual assets such as icons and Helsinki logo
+- Design token documentation
+- Component documentation
+- About section with news, release notes and road map
+- Resources section
+- Contribution guidelines
+- Form examples and patterns
+
+## Getting started
+
+When making changes to the documentation website, the recommended way is to get it running locally. This way you can review and see your changes in real-time like they would show up in production.
+
+### Setting up the local development environment
+
+1. Clone the HDS repository.
+```
+git clone https://github.com/City-of-Helsinki/helsinki-design-system.git
+```
+
+2. Go to the root of the project and install dependencies with `yarn`.
+```
+cd helsinki-design-system
+yarn
+```
+
+3. Build packages with `yarn`. Note, you need to build all HDS packages since the documentation site uses those as well.
+```
+yarn build
+```
+
+4. Start the development server. It will watch for file changes automatically.
+```
+cd new-site
+yarn start
+```
+
```

Comment:
If we want to improve the guidance, either remove the yarn start from just above or add a subtitle before the step, saying "If starting the new docsite fails, install missing local dependencies".